src/sass/_normalize.scss

```diff
@@ -0,0 +1,350 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
```

Comment:
This is a super good touch to ensure this works across all browsers. I was unsure how to implement this, but now I know. Good work Jeff!